common/common_functions.sh

```diff
@@ -355,19 +355,38 @@ function deploy_calico {
 }
 
 function create_macvlan_net {
-    local macvlan_file="${ARTIFACTS}/macvlan-net.yaml"
+    local macvlan_file="${1:-$ARTIFACTS/macvlan-net.yaml}"
```

Comment:
are the changes in create_macvlan_net related to the PR ?

its not mentioned in the commit message

Reply:
@adrianchiris This will be removed in favor of the udev rule